doc/panes/editor.rst

```diff
@@ -2,21 +2,30 @@
 Editor
 ######
 
-Spyder's multi-language **Editor** integrates a number of powerful tools right out of the box for an easy to use, efficient editing experience.
-The Editor's key features include syntax highlighting (``pygments``); real-time code and style analysis (``pyflakes`` and ``pycodestyle``); on-demand completion, calltips and go-to-definition features (``rope`` and ``jedi``); a function/class browser, horizontal and vertical splitting, and much more.
+Spyder's multi-language **Editor** pane is the key element of the IDE, where you can create, open, and modify source files.
+The Editor offers a variety of core features, such as autocompletion, real-time analysis, syntax highlighting, horizontal and vertical splitting, and much more.
+In addition, it integrates a number of powerful tools for an easy to use, efficient editing experience.
 
 .. image:: /images/editor/editor-standard.png
    :alt: Spyder's Editor panel, split horizontally and with style analysis
 
-:guilabel:`Outline Explorer` (function/class/method browser) and horizontal/vertical splitting capabilities:
+==============
+Key components
+==============
 
-.. image:: /images/editor/editor-outline-standard.png
-   :alt: Spyder outline panel, showing the functions/classes/methods in a file
+The Editor pane consists of the following areas:
 
-Real-time code and style analysis with ``pyflakes`` and ``pycodestyle``:
+.. image:: /images/editor/editor-components.png
+   :alt: Spyder's Editor pane, showing its different areas (described below)
 
-.. image:: /images/editor/editor-inset-code-analysis.png
-   :alt: A snippit of code in the Spyder Editor, showing code style warnings
+1. The left sidebar shows line numbers and displays any code analysis errors/warnings that exist in the current file.
+   Clicking a line number selects the code/text on that line.
```

Comment:
Should we mention here that users can set breakpoints by doing a right-click to the right of line numbers?

Reply:
(See discussion below)